web/src/features/charts/bar-breakdown/BarBreakdownChart.tsx

```diff
@@ -138,8 +150,93 @@ function BarBreakdownChart({
           isMobile={false}
         />
       )}
+      <div className="pt-2">
+        <Accordion title={t('data-sources.title')} className="text-md">
+          <div>
+            {currentZoneDetail?.capacitySources && (
+              <Source
+                title="Installed capacity data"
+                icon={
+                  <div
+                    className={`mt-[2px] h-[16px] w-[16px]  bg-[url('/images/utility-pole_light.svg')] bg-center dark:bg-[url('/images/utility-pole_dark.svg')]`}
+                  />
+                }
+                sources={GetSourceArrayFromDictionary(currentZoneDetail?.capacitySources)}
+              />
+            )}
+            {currentZoneDetail?.source && (
+              <Source
+                title="Power generation data"
+                icon={
+                  <div
+                    className={`mt-[1px] h-[16px] w-[16px] bg-[url('/images/wind-turbine_light.svg')] bg-center dark:bg-[url('/images/wind-turbine_dark.svg')]`}
+                  />
+                }
+                sources={[currentZoneDetail?.source]}
+              />
+            )}
+            {emissionData && (
+              <Source
+                title="Emission factor data"
+                icon={
+                  <div
+                    className={`mt-[1px] h-[16px] w-[16px] bg-[url('/images/industry_light.svg')] bg-center dark:bg-[url('/images/industry_dark.svg')]`}
+                  />
+                }
+                sources={emissionData}
+              />
+            )}
+          </div>
+        </Accordion>
+      </div>
     </div>
   );
 }
 
 export default BarBreakdownChart;
+
+function GetSourceArrayFromDictionary(sourceDict: {
+  [key in ElectricityModeType]: string[] | null;
+}): string[] {
+  const sourceArray: string[] = [];
+  if (sourceDict == null) {
+    return sourceArray;
+  }
+  for (const key of Object.keys(sourceDict)) {
+    const capacitySource = sourceDict?.[key as ElectricityModeType];
+    if (capacitySource != null) {
+      for (const source of capacitySource) {
+        if (!sourceArray.includes(source)) {
+          sourceArray.push(source);
+        }
+      }
+    }
+  }
+  return sourceArray;
+}
+
+function Source({
+  title,
+  icon,
+  sources,
+}: {
+  title: string;
+  icon: React.ReactNode;
+  sources: string[];
+}) {
+  return (
+    <div className="flex flex-col py-2">
+      <div className="flex flex-row pb-2">
+        <div className="mr-1">{icon}</div>
+        <div className="text-md font-semibold">{title}</div>
+      </div>
+      <div className="flex flex-col gap-2">
+        {sources.map((source, index) => (
+          <div key={index} className="text-xm">
+            {source}
+          </div>
+        ))}
+      </div>
```

Comment:
This should probably have some indentation to clearly mark it as a group of sources. @Alportan what do you think?

![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/463c4c46-2cd0-467d-ab3b-94461e53aa74)

Reply:
I agree. Should I change it to have indentation @Alportan ?

Reply:
Hmmm, I did played a bit with indentation when I was designing, but (at least on mobile 📱) it takes alot of screen estate to indent them especially for longer sources which can get into 2+ rows. 

I would still keep it left aligned and maybe increase the padding between the sections a bit. What do you think about that? @VIKTORVAV99 @silkeholmebonnen

Reply:
Let me play a bit and get back to you!

Reply:
Increasing the padding comes with its own negative effects (also mostly for mobile) with showing less information on the vertical space instead.

Your concerns are very valid though so maybe we should leave this as is for now and test out both? Could even have something more conditional based on screen size where  small screens have no indentation and somewhat larger screen does (think iPhone vs Iphone Plus or along those lines).

Reply:
I'll add a small task to revise the source alignment to left aligned, with the icon leading, as we're heading towards that direction anyways with the new design. 🙌